packages/react-reconciler/src/ReactFiberPerformanceTrack.js

```diff
@@ -658,11 +659,13 @@ export function logBlockingStart(
           // $FlowFixMe[method-unbinding]
           console.timeStamp.bind(
             console,
-            isSpawnedUpdate
-              ? 'Cascading Update'
-              : renderStartTime - updateTime > 5
-                ? 'Update Blocked'
-                : 'Update',
+            isPingedUpdate
+              ? 'Promise Resolved'
```

Comment:
Not related to these changes, just thinking out loud.

"Promise Resolved" sounds like an instant event, just a timestamp, instead of a complete event, where you have start and end.

I can't really think of a better naming for this, just want to avoid users thinking that Scheduler was actually waiting for this Promise to be resolved and this was blocking work. Same can be said about Cascading Update, though.

What do you think about naming this something like "Idle... Promise Resolved" and "Idle... Cascading Update"?